js/libs/builder.js

```diff
@@ -686,6 +629,47 @@ var app = new Vue({
 
       return string;
     },
+    initDataSourceProvider: function() {
+      var $vm = this;
+      var dataSourceData = {
+        dataSourceTitle: 'Form builder data source',
+        dataSourceId: $vm.settings.dataSourceId,
+        appId: Fliplet.Env.get('appId'),
+        default: {
+          name: 'Form builder data for ' + Fliplet.Env.get('appName'),
+          definition: {
+            'guid': 'Column 1'
+          },
+          entries: [],
+          columns: []
+        },
+        accessRules: [
```

Comment:
Incorrect access rules. See https://docs.google.com/presentation/d/1vvEvfGm_XGjKE3vp1duMy8DuqrgQGD3Dvlf4M-xNQQU/edit?ts=5f341a02#slide=id.g8d5e20b4f1_0_0

@upplabs-alex-levchenko @sofiiakvasnevska Please follow the specs provided. I shouldn't need to open the document to check this myself.

Reply:
@tonytlwu it said in the spec that form builder rules are: `Write, Update - only if the update is enabled` and that is exactly how rules are set. Is there something more that should be added to these rules?
![image](https://user-images.githubusercontent.com/53430352/93514139-84804780-f92f-11ea-8f73-938c22b83279.png)

Reply:
@YaroslavOvdii Yes, that is correct. Sorry this was my bad, I saw `"allow": "all"` and it looked like we're allowing them to do everything, but I was mistaken. This is fine.